http/handler.go

```diff
@@ -350,7 +350,7 @@ func wrapGenericHandler(core *vault.Core, h http.Handler, props *vault.HandlerPr
 			return
 		}
 
-		h.ServeHTTP(w, r)
+		h.ServeHTTP(&logical.NamespaceResponseWriter{ResponseWriter: w, NamespacePath: r.Header.Get("X-Vault-Namespace")}, r)
```

Comment:
I'd rather see ResponseWriter and NamespacePath on their own lines, e.g.

```
h.ServeHTTP(&logical.NamespaceResponseWriter{
  ResponseWriter: w, 
  NamespacePath: r.Header.Get("X-Vault-Namespace"),
}, r)
```

This is more in line with our formatting conventions, and to me it looks more readable.

Reply:
Will do! Thanks.